### DISCUSSION TITLE
Someone help me out. Invalid JSON error

### DISCUSSION THREAD

**Discussion by Despicable-Games** [Epicpkmn11/WordleDS#24] [Support and Suggestions] [answered]
whenever I try to update the word list on a good Wi-Fi connection it always pops up with "invalid JSON" does anyone know what this means? if so, can somebody help me with fixing the error?

**Comment by Epicpkmn11** [comment, marked as answer]
If this happened yesterday, my website was messed up for a bit. It looks to be serving correctly right now though so I'm not sure why it'd be happening now...
